### DISCUSSION TITLE
ColorKt.copyColor can only be called from the within the same library group

### DISCUSSION THREAD

**Discussion by zell-mbc** [patrykandpatrick/vico#862] [Questions] [answered]
### Question

I have been catching up with my app on the several alpha releases one by one until I reached 2.0.0-alpha.28, during one of the upgrades a new warning got introduced which I don't see mentioned in any of the release notes.
The `rememberMarker` sample code I took from your examples has got the following line 

`rear = ShapeComponent(color.copyColor(alpha = 0.15f), Shape.Pill),`
which now produces the warning below:

![image](https://github.com/user-attachments/assets/c011969a-0753-4266-8787-9f9b011731e4)

I can obviously suppress the warning but was wondering if I should or if the `rememberMarker` sample code would need an update?

### Vico version(s)

2.0.0-alpha.28

### UI framework(s)

Jetpack Compose

**Comment by patrickmichalik** [comment, marked as answer]
Hello! This is an oversight on our part. Thanks for bringing it to our attention. `rememberMarker` will be updated to use `ColorUtils.setAlphaComponent` instead of Vico’s restricted `Int.copyColor`, and we suggest that you make the same change on your end.

**Comment by patrickmichalik** [comment]
This has been addressed with [Vico 2.0.0 Beta 2](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-beta.2). Cheers!
